backend/src/graphql/resolver/CommunityResolver.test.ts

```diff
@@ -92,7 +101,9 @@ describe('CommunityResolver', () => {
               {
                 id: 3,
                 foreign: homeCom3.foreign,
-                publicKey: expect.stringMatching('publicKey-HomeCommunity'),
+                publicKey: expect.stringMatching(
+                  '2ca593275aa4c11f9c3d43cd4d39586c70e2b7f4359739381940b62d1c8e8928',
+                ),
```

Comment:
Why not using the variable `homeCom3.publicKey` instead of hardcoded string?

Reply:
Community.publicKey should be a Buffer type, graphql returns a hex string.
But you right I could make it way better. Save for future test, if the key will be actually validated.

Reply:
good idea